megatron/arguments.py

```diff
@@ -269,12 +269,13 @@ def validate_args(args, defaults={}):
         args.ffn_hidden_size = 4 * args.hidden_size
 
     if args.swiglu:
-        # reduce the dimnesion for MLP since projections happens on
-        # two linear layers. this keeps the number of paramters in
-        # the same ballpark as the counterpart with 4*h size
-        # we keep it a multiple of 64, which means the actual tensor size
-        # will be a multiple of 64 / tp_size
-        args.ffn_hidden_size = int((4 * args.hidden_size * 2 / 3) / 64) * 64
+        if args.ffn_hidden_size is None:
```

Comment:
I think we just need one if condition as `if args.swiglu and args.ffn_hidden_size is None:`